app/views/tag/show/_tab_content.html.erb

```diff
@@ -1,40 +1,6 @@
 <div class="tab-content" style="clear:both;">
   <!-- separate these in partials-->
   <!-- use controller to determine which partial to render -->
-  <% if @node_type == "note" %>
-    <div class="tab-pane active" id="notes">
```

Comment:
Aha - ok, so we do need to delete line 4, but lines 5-12 need to be preserved so that we still show /something/ -- does that make sense? We can also remove the comments on lines 2 and 3!